test/application/widgets/theme/memo_terminal_test.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:memo/application/constants/strings.dart' as strings;
+import 'package:memo/application/widgets/theme/custom_button.dart';
+import 'package:memo/application/widgets/theme/destructive_button.dart';
+import 'package:memo/application/widgets/theme/memo_terminal.dart';
+import 'package:memo/application/widgets/theme/rich_text_field.dart';
+import 'package:mocktail/mocktail.dart';
+
+import '../../../utils/fakes.dart';
+import '../../../utils/widget_pump.dart';
+
+void main() {
```

Comment:
IMO, could be tested, up to you tho:

- `onRemove` is not called when tapping the widget and it's `null`.
- "private" controllers are created for those that aren't provided.

Reply:
> onRemove is not called when tapping the widget and it's null.

I'm not sure what you suggested here. What do you mean with `and it's null.`?

> "private" controllers are created for those that aren't provided.

I removed the controllers initialization because they're not used in `MemoTerminal`, although the test that you suggested would fit will in `CustomTextField`, `RichTextField` and `TagsField`. What do you think?